src/wristpy/processing/calibration.py

```diff
@@ -201,6 +217,7 @@ def _chunked_calibration(
             CalibrationError: If all possible chunks have been used and the calibration
                 process fails to get below the `min_calibration_error` threshold.
         """
+        logger.debug("running chunked calibration.")
```

Comment:
Minor thing; but you're mixing capitalized and non-capitalized logging statements. I'd suggest making a consistent choice here.